request.rest

```diff
@@ -0,0 +1,27 @@
+### Add a new comment
+POST http://localhost:3000/comment
+Content-Type: application/json
+
+{
+  "postId": "<post_id>",
+  "content": "This is a comment",
+  "author": "67890"
```

Comment:
I think it would look better with your name for exmaple or something more meaningful